### PR TITLE
[Integration] MySQL: deprecate _insert (use insertFields)

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Core:
CHANGE: MySQL: use insertFields instead of _insert 

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Please review.

**Comment by fstagni** [comment]
Please do review.

File: DataManagementSystem/DB/FileCatalogComponents/test/Test_DataManagement_FileCatalogComponents.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+""" Test class for FileCatalogComponents
```
Comment (chaen) [inline DataManagementSystem/DB/FileCatalogComponents/test/Test_DataManagement_FileCatalogComponents.py]: Dont' you want to remove that ? It seems useless no ? I thought it was just to try to make codecov happy.

File: WorkloadManagementSystem/DB/JobDB.py [hunk 2]
```diff
@@ -750,7 +750,7 @@ def setJobStatus(self, jobID, status='', minor='', application=''):
       return ret
     application = ret['Value']
 
-    cmd = 'REPLACE JobsStatus (JobID,Status,MinorStatus,ApplicationStatus) VALUES (%d,%s,%s,%s)' % (
+    cmd = 'REPLACE INTO JobsStatus (JobID,Status,MinorStatus,ApplicationStatus) VALUES (%d,%s,%s,%s)' % (
```
Comment (chaen) [inline WorkloadManagementSystem/DB/JobDB.py]: For completeness: `REPLACE INTO` is not SQL standard. standard would be `insert.. on duplicate key update`

File: DataManagementSystem/DB/FileCatalogComponents/test/Test_DataManagement_FileCatalogComponents.py (hunk 1 above)
Comment (fstagni) [reply DataManagementSystem/DB/FileCatalogComponents/test/Test_DataManagement_FileCatalogComponents.py]: It is not super useful but at least it covers few lines of code, and clearly checks it there are no circular imports. So, I would leave it.

File: WorkloadManagementSystem/DB/JobDB.py (hunk 2 above)
Comment (fstagni) [reply WorkloadManagementSystem/DB/JobDB.py]: I know... but is it a problem? Anyway we are on MySQL only.

Comment (chaen) [reply WorkloadManagementSystem/DB/JobDB.py]: no no, I am just saying, since we regularly get the question..

**Pull request merged**